imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -477,64 +475,34 @@ def process_swapi_science(sci_dataset):
     dataset["swp_pcem_counts"] = xr.DataArray(
         np.array(swp_pcem_counts, dtype=np.uint16),
         dims=["epoch", "energy"],
-        attrs=dataclasses.replace(
-            counts_attrs,
-            fieldname="Primary CEM counts",
-            label_axis="PCEM cnts",
-            catdesc="Primary Channel Electron Multiplier (CEM) counts",
-        ).output(),
+        attrs=metadata["swp_pcem_counts"],
     )
     dataset["swp_scem_counts"] = xr.DataArray(
         np.array(swp_scem_counts, dtype=np.uint16),
```

Comment:
I added a DATA_TYPE column to my spreadsheet thinking it could be used to specify the `dtype` when creating the `DataArray`s.

Reply:
I believe it could be accessed like `dtype=np.dtype(metadata["swp_scem_counts"]["DATA_TYPE"])` if you wanted to set that here in that way.

One thing that leads to is that with the "dump everything into metadata" approach, there would be a lot of extra metadata attributes then if we care about that showing up in the output data files or not. Something worth thinking about whether we'd want to pop it off the dict at some point, or just live with it also being an attribute field.